src/styles/components/_feedback.scss

```diff
@@ -0,0 +1,17 @@
+@import '../settings/colors';
+
+#w-feedback__dropdown {
```

Comment:
I would suggest using classes here. Not a huge deal that we're using IDs but I also don't see much value in it. If (for whatever reason) we did have more than one feedback element on a page then this would technically be wrong (though I think it would still work).

Reply:
I'll spare you the details unless helpful but I could imagine more than one feedback component on a page